### PR TITLE
2318 apiv3 frontend upload progress

### PR DESCRIPTION
**Pull request by aviveiros11**
#### Any background context you want to provide?
Part of the effort to convert the frontend to use API v3

#### What's this PR do?
See attached issue.

#### How should this be manually tested?
Verify checks in attached issue are warranted.

Also check the following:
- [x] Make sure `file upload` progress bar behavior remains the same
- [x] Make sure `mapping` progress bar behavior remains the same
- [x] Make sure `confirm mapping & start matching` progress bar behavior remains the same

#### What are the relevant tickets?
#2318

### CODE REVIEW

File: seed/views/v3/uploads.py [hunk 1]
```diff
@@ -59,7 +58,7 @@ class UploadViewSet(viewsets.ViewSet):
             ),
             AutoSchemaHelper.form_string_field(
                 name='source_type',
-                required=True,
+                required=False,
```
Comment (adrian-lara) [inline seed/views/v3/uploads.py:61]: I would've thought that source_type should still be required? Though admittedly, I don't know why we're handling the "missing" case when treating the request as POST by checking for source_type when treating the request as a GET.
`source_type = request.POST.get('source_type', request.GET.get('source_type'))`

File: seed/views/v3/uploads.py [hunk 2]
```diff
@@ -46,7 +46,6 @@ class UploadViewSet(viewsets.ViewSet):
     @swagger_auto_schema(
         request_body=no_body,
         manual_parameters=[
-            AutoSchemaHelper.query_org_id_field(),
```
Comment (adrian-lara) [inline seed/views/v3/uploads.py:49]: I think we want to undo these changes and update the frontend to pass in an org_id for this request.

File: seed/static/seed/js/directives/sdUploader.js [hunk 3]
```diff
@@ -17,10 +17,11 @@
  */
 
 var makeFileSystemUploader = function (scope, element, allowed_extensions) {
+  console.log(scope);
```
Comment (adrian-lara) [inline seed/static/seed/js/directives/sdUploader.js:20]: 😬 Note to remove.

File: seed/views/v3/uploads.py [hunk 4]
```diff
@@ -103,12 +103,11 @@ def create(self, request):
         with open(path, 'wb+') as temp_file:
             for chunk in the_file.chunks():
                 temp_file.write(chunk)
-        org_id = request.query_params.get('organization_id', None)
+
         import_record_pk = request.POST.get('import_record', request.GET.get('import_record'))
         try:
             record = ImportRecord.objects.get(
                 pk=import_record_pk,
-                super_organization_id=org_id
```
Comment (adrian-lara) [inline seed/views/v3/uploads.py:111]: We should undo these code changes too.

**Comment by adrian-lara** [comment]
For some reason, I'm having issues Updating With BuildingSync. Can you try to reproduce on the inventory detail page? I have a feeling it might just be user error 😬

**Comment by aviveiros11** [comment]
@adrian-lara I think this is an unrelated bug? I was able to reproduce this on both my feature branch as well as develop.  It looks like when updating with building sync on the inventory detail page it actually hits `http://0.0.0.0:8000/api/v2.1/properties/2122/update_with_building_sync/?cycle_id=29&organization_id=12` instead of the upload path.
